typings/index.d.ts

```diff
@@ -3148,6 +3148,22 @@ declare module 'discord.js' {
 
   type SystemChannelFlagsResolvable = BitFieldResolvable<SystemChannelFlagsString>;
 
+  type SystemMessageType =
+    | 'RECIPIENT_ADD'
+    | 'RECIPIENT_REMOVE'
+    | 'CALL'
+    | 'CHANNEL_NAME_CHANGE'
+    | 'CHANNEL_ICON_CHANGE'
+    | 'PINS_ADD'
+    | 'GUILD_MEMBER_JOIN'
+    | 'USER_PREMIUM_GUILD_SUBSCRIPTION'
+    | 'USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_1'
+    | 'USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_2'
+    | 'USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_3'
+    | 'CHANNEL_FOLLOW_ADD'
+    | 'GUILD_DISCOVERY_DISQUALIFIED'
+    | 'GUILD_DISCOVERY_REQUALIFIED';
```

Comment:
```suggestion
  type SystemMessageType = Exclude<MessageType, 'DEFAULT' | 'REPLY'>;
```
maybe this would make more sense? however it's inconsistent with the actual exported constant then

Reply:
This is a good point, the export should be exclusionary as well, making it consistent.